spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -296,6 +296,10 @@ private DepInterpreter getDepInterpreter() {
     return (DepInterpreter) p;
   }
 
+  private boolean isYarnMode() {
+    return getProperty("master").equals("yarn-client") || getProperty("master").equals("yarn");
```

Comment:
Yes, you're right. It's because Zeppelin don't support `yarn-cluster`yet.
You think it's better to include `yarn-cluster` too?